implemented all changes from court feedback

raminabhan opened the pull request:
- updated intro screen language
- added hyperlink for searching docket numbers
- removed "pains" from attorney message
- removed "you can pick more than one reason" from eviction reason screen
- removed "allegedly" from reason page answer choices
- updated off ramp screen language
- removed "keep going anyway" button on off ramp
- updated language on eviction property address screen
- updated language on landlord name screen
- added help info box on landlord name screen
- removed "service of process" language on notice screen
- updated language on review/download form screen
- updated language on other recent evictions screen
- updated language on evictions for fault reason screen
- removed bolded language on satisfaction of judgment screen

nonprofittechy commented:
@rami can you link the issues that this will close? 

![image](https://github.com/user-attachments/assets/9a3f4db0-8423-4b59-8aef-a7e27c5de88a)

raminabhan commented:
all set. might be a couple others but it only allowed me to link 10. i believe we also have some repeat issues.

nonprofittechy commented on docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml:
```diff
@@ -253,27 +253,26 @@ subquestion: |
-  * Help you understand if you can seal your eviction records.
-  * Make a "Petition to Seal eviction" for you to file with the court.
+  * Help you understand if you can seal your eviction record using this interview.
+  * Fill-out a Petition to Seal Eviction Record Pursuant to G.L. c. 239, § 16 to file with the court.
```
```suggestion
  * Fill out a Petition to Seal Eviction Record Pursuant to G.L. c. 239, § 16 to file with the court.
```

nonprofittechy commented on docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml:
```diff
@@ -253,27 +253,26 @@ subquestion: |
-  1. Information about the reason for your eviction
-  1. The date of the **final decision** (judgment) in your eviction action
-  1. Information about any other eviction cases that you have had since the case you are trying to seal.
+  1. The <a href="https://www.mass.gov/info-details/how-to-search-court-dockets?_gl=1*1t4lg0v*_ga*Mjc1NTYwNDQxLjE3NDE2MjkwMjA.*_" style="text-decoration: none; color: green; font-weight: bold;">docket number</a> of the eviction case you want to seal
```
```suggestion
  1. The docket number of the eviction case you want to seal (<a href="https://www.mass.gov/info-details/how-to-search-court-dockets?_gl=1*1t4lg0v*_ga*Mjc1NTYwNDQxLjE3NDE2MjkwMjA.*_">find your docket number in a new tab</a>)
```

1. Need to warn about external links
2. Links should always use the same style. Since this opens in a new tab, green link style isn't appropriate.

(did the court mean to make this a reference term, like the "green word" on the front page?) https://docassemble.org/docs/initial.html#terms

If that's the case, we just need to add a definition of Docket Number, not an external link.

raminabhan replied:
Agree, I just went with their styling preference but it did seem counter intuitive to make it green. However, their instructions were to make it a hyperlink.

nonprofittechy replied:
Something like "it's a combination of letters and numbers uniquely identifying your case. It should be written on the paperwork you got from the court or your landlord"

nonprofittechy commented on docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml:
```diff
@@ -341,26 +338,24 @@ content: |
-  You can file the petition anyway, but no boxes that qualify you for sealing are checked. The court may reject your petition.
+  Please contact your Clerk's Office if you have additional questions.
 
-  If you have any questions, you can contact a lawyer or legal services organization for help.
 sets: continue_inapplicable
 buttons:
-  - Keep going anyway:
-      code: |
-        continue_inapplicable = True
```
Want to make sure we correctly understood the court's feedback here. Do they no longer want to let people keep going to file an incomplete petition? I'll ask today on our call.

nonprofittechy commented on docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml:
```diff
@@ -432,9 +427,9 @@ id: Fault Eviction
   Do you have any other recent eviction cases?
 subquestion: |
-  The judge needs to know about evictions in the last **7 years**.
+  Has a landlord filed a new eviction case against you at any time in the **LAST 7 YEARS?**
```
```suggestion
  Has a landlord filed a new eviction case against you at any time in the **last 7 years?**
```

We never use all caps for emphasis. See: https://assemblyline.suffolklitlab.org/docs/style_guide/formatting#only-use-all-capital-letters-for-acronyms-and-initialisms

nonprofittechy commented on docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml:
```diff
@@ -444,9 +439,9 @@ subquestion: |
+  - "Any fault reason: broke the lease other than owing rent.": prior_fault_action_139
     datatype: yesnoradio
-  - "139, §19: an emergency eviction for criminal activity": prior_139_action
+  - "139, §19: broke the law and the landlord filed an emergency eviction": prior_139_action
```
Same as above, this isn't grammatical anymore. Who broke the law?

nonprofittechy commented on docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml:
```diff
@@ -541,7 +535,7 @@ sets:
 id: tenancy_address
 question: |
-  Where did you live when the landlord filed the eviction?
+  What was the address of the property where the eviction occured?
```
Why is this passive voice?

raminabhan replied:
maybe "where _is_ the address of the property?" again, just went off of their feedback doc.

nonprofittechy replied:
Thought: if you had moved out before the eviction, it might not be the same address as where you were evicted from. Want to clarify it's the eviction address.

nonprofittechy replied:
@raminabhan I think this is the only blocking change. can you edit?

nonprofittechy commented on docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml:
```diff
@@ -915,9 +915,11 @@ confirm: True
 event: petition_to_seal_eviction_download
 question: |
-  All done
+  Your petition information has been filled out.
```
```suggestion
  Your petition information has been filled out
```

nonprofittechy commented on docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml:
```diff
@@ -915,9 +915,11 @@ confirm: True
 subquestion: |
   Thank you ${users}. Your form is ready to download and deliver.
+
+  **Do not close this page or all your progress will be lost.**
```
This is not true. Should we clarify with the court?

nonprofittechy replied:
Maybe just encourage people to make an account?

nonprofittechy replied:
From MATC feedback - they just want to make sure people don't think they are done yet and that they know there is a step to take.

1. We can change this to a recommendation for people to create an account.
2. I raised thought that we don't have to have an "efile" button but instead could automatically e-file for folks on the last screen, which I think they will consider.

nonprofittechy reviewed (changes requested):
See specific in-line comments and suggested changes. Some need to be clarified with the court.

raminabhan commented:
I would discuss with the court about the changes implemented. All of the changes I made were verbatim from the court's feedback document.

nonprofittechy commented on docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml:
```diff
@@ -444,9 +439,9 @@ subquestion: |
   for one of these reasons?
 fields:
-  - "Any fault reason: breaking the lease for a reason other than nonpayment of rent.": prior_fault_action_139
+  - "Any fault reason: broke the lease other than owing rent.": prior_fault_action_139
```
MATC agreed and will review

nonprofittechy commented:
@raminabhan please change the passive voice one to suggested language. We will get new revision on the other ungrammatical language, probably by end of day. That can be in a new PR though, so once the first is fixed I think this can be merged. Just request a new review when you've change PV.

Make sure you pull this branch into a new Docassemble project first to get the changes i just made.

raminabhan commented:
Will do!

raminabhan commented:
Should be ready to merge now. Changed the passive language, slightly reworded the "broke the lease" language, made the "last 7 years" back to lowercase, and changed the 'C' in Court paperwork to lower case.

raminabhan commented:
Ah okay, understood.
